### PR TITLE
Improve soroban setup instructions.

### PR DESCRIPTION
**Pull request by fnando**

### CODE REVIEW

**Comment by fnando** [comment]
I'm not sure what's up with prettier. If I run it locally, this is what I get:

```console
$ prettier -c docs/smart-contracts/getting-started/setup.mdx
Checking formatting...
[warn] jsxBracketSameLine is deprecated.
All matched files use Prettier code style!
```

File: docs/smart-contracts/getting-started/setup.mdx [hunk 1]
```diff
@@ -27,13 +30,29 @@ To build and develop contracts you need only a couple prerequisites:
...
+
+The soroban CLI uses emojis. To properly render them on Windows, we recommend using [Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/). See [how to install Windows Terminal](https://learn.microsoft.com/en-us/windows/terminal/install) on Microsoft Learn.
+
+:::
+
+If you're already using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install), you can also follow the instructions for Linux.
+
+### Other
+
+For other methods of installing [Rust], check out: https://www.rust-lang.org/tools/install
```
Comment (leighmcculloch) [inline docs/smart-contracts/getting-started/setup.mdx:55]: Small tweak to remove language we'll use elsewhere when referring to git operations.
```suggestion
For other methods of installing [Rust], see: https://www.rust-lang.org/tools/install
```

File: docs/smart-contracts/getting-started/setup.mdx [hunk 2]
```diff
@@ -168,13 +201,7 @@ soroban keys address alice
...
 Like the Network configs, the `--global` means that the identity gets stored in `~/.config/soroban/identity/alice.toml`. You can omit the `--global` flag to store the identity in your project's `.soroban/identity` folder instead.
 
-All this did so far is generate a public/private keypair on your local machine. No network requests were made, and `alice` has no funds on Testnet. This means that you can't make any transactions with `alice` yet.
-
-To get `alice` some Testnet tokens, you'll need to use [Friendbot](https://developers.stellar.org/docs/fundamentals-and-concepts/testnet-and-pubnet#friendbot). All Stellar and Soroban test networks have a Friendbot that you can use to get some test tokens. The public Friendbot instance for Testnet lives at `https://friendbot.stellar.org`. The cli includes a `fund` subcommand to fund your address with Friendbot:
-
-```bash
-soroban keys fund alice --network testnet
-```
+By default, `soroban keys generate` will fund your account using [Friendbot](https://developers.stellar.org/docs/fundamentals-and-concepts/testnet-and-pubnet#friendbot). If you don't want this behavior, run this command with `--no-fund`.
```
Comment (leighmcculloch) [inline docs/smart-contracts/getting-started/setup.mdx:204]: Small tweak to reduce the chance this is misinterpreted to mean you need to run an additional command with the option:
```suggestion
By default, `soroban keys generate` will fund your account using [Friendbot](https://developers.stellar.org/docs/fundamentals-and-concepts/testnet-and-pubnet#friendbot). If you don't want this behavior, append `--no-fund` to the command when running it.
```

**Review by leighmcculloch — APPROVED** [review]
Couple suggested tweaks inline, regardless lgtm.

File: docs/smart-contracts/getting-started/setup.mdx [hunk 3]
```diff
@@ -140,12 +159,26 @@ Soroban has a test network called Testnet that you can use to deploy and test yo
...
   --rpc-url https://soroban-testnet.stellar.org:443 \
   --network-passphrase "Test SDF Network ; September 2015"
 ```
 
+  </TabItem>
+  <TabItem value="windows" label="Windows">
+
+```bash
+soroban network add --global testnet --rpc-url https://soroban-testnet.stellar.org:443 --network-passphrase "Test SDF Network ; September 2015"
+```
```
[the remark this answers is not in the document]
Comment (leighmcculloch) [reply docs/smart-contracts/getting-started/setup.mdx:177]: Actually, quick thought: We should be able to use the `^` on windows for line continuation. I don't have windows setup to test it, but I know you do. Worth checking if we can preserve the multiline view for easy reading. But we'd need to check it on cmd.exe, powershell, and windows terminal I guess. If they don't all work the same way, then yeah lets keep it as is.

**Review by leighmcculloch — APPROVED** [review]

**Review by leighmcculloch — APPROVED** [review]

**Pull request merged**